### PR TITLE
MaxPool-8 python API

### PR DESCRIPTION
**Pull request by tomdol**
### Details:
 - MaxPool-8 exposed to python
 - Tests adjusted to the new op with 2 outputs
 - this PR depends on https://github.com/openvinotoolkit/openvino/pull/7115 

### Tickets:
 - *54670*

### CODE REVIEW

File: ngraph/python/src/ngraph/opset8/ops.py [hunk 1]
```diff
@@ -276,3 +276,57 @@ def gather(
...
+                                 over the axes.
+    @param  dilations:           The dilation of filter elements(distance between elements).
+    @param  pads_begin:          The number of pixels to add at the beginning along each axis.
+    @param  pads_end:            The number of pixels to add at the end along each axis.
+    @param  kernel_shape:        The pooling operation kernel shape.
+    @param  rounding_type:       Determines used rounding schema when computing output shape.
+                                 Acceptable values are: ['floor', 'ceil']
+    @param  auto_pad:            Determines how the padding is calculated. Acceptable values:
+                                 [None, 'same_upper', 'same_lower', 'valid']
+    @param  index_element_type:  The data type used for the indices outputof this operator.
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/opset8/ops.py:308]: You can add a note of default value `i64`. Moreover should it be `output of` with a space between words?

File: ngraph/python/src/ngraph/opset8/ops.py [hunk 2]
```diff
@@ -276,3 +276,57 @@ def gather(
...
+    @param  axis:                The first dimension in the data shape used to determine the maximum
+                                 returned index value. The value is the product of all dimensions
+                                 starting at the provided axis.
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/opset8/ops.py:311]: Also would be good to have a note about default value `0`.

File: ngraph/python/src/ngraph/opset8/ops.py [hunk 3]
```diff
@@ -276,3 +276,57 @@ def gather(
...
+        [as_node(data)],
+        {
+            "strides": strides,
+            "dilations": dilations,
+            "pads_begin": pads_begin,
+            "pads_end": pads_end,
+            "kernel": kernel_shape,
+            "rounding_type": rounding_type.upper(),
+            "auto_pad": auto_pad.upper(),
+            "index_element_type": index_element_type,
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/opset8/ops.py:329]: Is `index_element_type` not depending on letters case or is it automatically casted using `lower()`/`upper()` ? If not please also specify that as in lines above.

**Review by jiwaszki — COMMENTED** [review]
Small changes to apply. Besides that LGTM 🚀

File: ngraph/python/src/ngraph/opset8/ops.py (hunk 3 above)
Comment (tomdol) [reply ngraph/python/src/ngraph/opset8/ops.py]: Good catch, it should be passed with lower case to cpp.

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**